docs/reference/mapping/date-format.asciidoc

```diff
@@ -211,6 +211,17 @@ date formatter in that case.
 |=======================================================================
 
 [float]
+[[strict]]
+=== Strict date formats
```

Comment:
updated the docs and added this for each date format, where I noticed that I missed one date format.